std-bits/table/src/main/java/org/enso/table/problems/ColumnAggregatedProblemAggregator.java

```diff
@@ -0,0 +1,34 @@
+package org.enso.table.problems;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.enso.table.data.table.problems.ColumnAggregatedProblem;
+
+/** A ProblemAggregator that is capable of merging ColumnAggregatedProblems. */
+public class ColumnAggregatedProblemAggregator extends ProblemAggregator {
+  public ColumnAggregatedProblemAggregator(ProblemAggregator parent) {
+    super(parent);
+  }
+
+  private final List<ColumnAggregatedProblem> aggregatedProblemList = new ArrayList<>();
+
+  public void reportColumnAggregatedProblem(ColumnAggregatedProblem problem) {
+    for (ColumnAggregatedProblem p : aggregatedProblemList) {
+      if (p.merge(problem)) {
+        // The problem was merged with an existing one.
+        return;
+      }
+    }
+
+    aggregatedProblemList.add(problem);
+  }
```

Comment:
`ColumnAggregatedProblemAggregator` is a bit of a remnant of the old logic.

We would probably have done it slightly differently with this new system. I think we could streamline its internal implementation in the future a little bit - to make it slightly more efficient.

But it currently works well and fits OK into the current architecture, so I thought there is no need to over-engineer this at this point, as I don't think it is a bottleneck by any means - if we find out it is, we can roll more specialized solutions to aggregate the `ColumnAggregatedProblem` more efficiently.